### DISCUSSION TITLE
Sign out link

### DISCUSSION THREAD

**Discussion by iainbeeston** [avo-hq/avo#792] [Feature requests]
It'd be really useful to have a sign out link in the navbar, maybe allowing a path to be passed in in the avo initializer so it supports different frameworks.

**Comment by iainbeeston** [comment]
It looks like it does automatically add this but only if your user model is called "user". If you have a user model with another name there is an undocumented config option: you have to set `config.current_user_resource_name = :my_model` in `config/initializers/avo.rb`. (I wonder why we need both `current_user_resource_name` and `config.current_user_method`?)

**Reply by adrianthedev** [reply]
I was just about to answer about `current_user_resource_name`. Odd that it's not documented. I'm on it!

The reason that there are two of them is that not everyone is using the `current_SOMETHING` method. You might use `Current.user` or something else and the `Avo.config.current_user_method` is actually a block that takes whatever you pass it and fetches the current user.

**Reply by adrianthedev** [reply]
Docs on that

https://docs.avohq.io/2.0/authentication.html#customise-the-sign-out-link

**Comment by iainbeeston** [comment]
(It isn’t related to this issue, but I think documentation about how user avatars and names are picked up automatically would be useful too!)

**Reply by adrianthedev** [reply]
So true! On it!

**Reply by adrianthedev** [reply]
🙌 https://docs.avohq.io/2.0/customization.html#customize-profile-name-photo-and-title

If you see other things missing in the docs, please keep them coming!
